src/main/java/easton/DateTimeFormatException.java

```diff
@@ -0,0 +1,7 @@
+package easton;
+
+public class DateTimeFormatException extends Exception {
```

Comment:
Perhaps this could be in another package such as the exceptions package?

Reply:
okay. I will move all the exception into an exception package